app/views/articles/show.html.erb

```diff
@@ -159,17 +159,20 @@
                   <em>with <%= @article.co_author_name_and_path.html_safe %></em>
                 <% end %>
 
-                <% if should_show_updated_on?(@article) %>
-                  <em>Updated on <time datetime="<%= @article.edited_at&.utc&.iso8601 %>"><%= @article.edited_at&.strftime("%b %d, %Y") %></time></em>
-                <% elsif should_show_crossposted_on?(@article) %>
+    <% if should_show_crossposted_on?(@article) %>
                   <em>
                     Originally published at
                     <a href="<%= @article.canonical_url || @article.feed_source_url %>" style="color:#1395b8"><%= get_host_without_www(@article.canonical_url || @article.feed_source_url) %></a>
                     <% if @article.crossposted_at %>
                       on
                       <time datetime="<%= (@article.originally_published_at || @article.published_at)&.utc&.iso8601 %>"><%= (@article.originally_published_at || @article.published_at)&.strftime("%b %d, %Y") %></time>
                     <% end %>
+                    <% if should_show_updated_on?(@article) %>
```

Comment:
That's a good point! I actually noticed that right as I was going to commit the initial changes, thinking at the very least the "Updated At" text should be extracted out instead of duplicating the code, but I got distracted by some issues I had with the pre-commit hooks (which I also think is why the formatting got all messed up on that one line... I'm not sure how that happened otherwise since the indentation looked fine in my editor at the time!)

One reason why I didn't go in this direction from the beginning is because I think it looks a bit weird to *just* show both crosspost and update text without any kind of separator between them, and in the original commit I chose parenthesis to separate them. For now to keep things simple I removed the parenthesis in my newest commit:
![2020-11-05-140504](https://user-images.githubusercontent.com/9145247/98296887-d4af7980-1f70-11eb-830b-161858b60b4e.png)

Reply:
You could extract it into a helper that does something like the following:

```ruby
originally_published_at = ... if ...
updated_at = ...if...

[originally_published_at, updated_at].compact.join(" | ")
```

This only adds the separator when both are non-nil:

```ruby
["something", nil].compact.join(" | ")
#=> "something"
[nil, "somethingelse"].compact.join(" | ")
#=> "somethingelse"
["something", "somethingelse"].compact.join(" | ")
#=> "something | somethingelse
```